lib/y2_installbase.pm

```diff
@@ -187,6 +187,42 @@ sub save_upload_y2logs {
     $self->investigate_yast2_failure() unless $args{skip_logs_investigation};
 }
 
+sub save_remote_upload_y2logs {
+    my ($self, %args) = @_;
+
+    return if (get_var('NOLOGS'));
+    $args{suffix} //= '';
+
+    # Do not test/recover network if collect from installation system, as it won't work anyway with current approach
+    # Do not recover network on non-qemu backend, as not implemented yet
+    $args{no_ntwrk_recovery} //= (get_var('BACKEND') !~ /qemu/);
+
+    # Try to recover network if cannot reach gw and upload logs if everything works
+    if (can_upload_logs() || (!$args{no_ntwrk_recovery} && recover_network())) {
```

Comment:
This won't work, `can_upload_logs` relies on `script_run`, so will always fail. Same with `recover_network`, so this condition will never be true.

Reply:
i removed totally the condition. if this is a concern let me know